bfasst/flows.py

```diff
@@ -134,7 +134,11 @@ def flow_ic2_lse_conformal(design, flow_args, build_dir):
     design.compare_golden_files_paths.append(design.path / design.top_file)
     design.compare_golden_files_paths.extend([design.path / f for f in design.get_support_files()])
     design.golden_is_verilog = design.top_is_verilog()
-    status = conformal_cmp(design, build_dir, flow_args)
+
+    gold_netlist = str(design.netlist_path)
+    rev_netlist = str(design.reversed_netlist_path.name)
+
+    status = conformal_cmp(design, build_dir, gold_netlist, rev_netlist, flow_args)
```

Comment:
Rather than create new variables (`gold_netlist` and `rev_netlist`), you can just pass in the paths directly as arguments:
```python
status = conformal_cmp(design, build_dir, design.netlist_path, design.reversed_netlist_path, flow_args)
```